renovate.json

```diff
@@ -0,0 +1,53 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base",
+    "schedule:daily",
+    ":dependencyDashboard",
+    ":semanticCommits",
+    ":semanticCommitScope(deps)",
+    "group:linters",
+    "group:googleapis",
+    "group:goOpenapi",
+    "group:reactMonorepo",
+    "group:reactrouterMonorepo"
+  ],
+  "packageRules": [
+    {
+      "datasources": ["docker"],
+      "updateTypes": ["patch"],
+      "groupName": "docker patch & minor updates"
+    },
+    {
+      "datasources": ["go"],
+      "updateTypes": ["patch", "minor"],
+      "groupName": "go.mod patch & minor updates"
+    },
+    {
+      "datasources": ["npm"],
+      "updateTypes": ["patch", "minor"],
+      "groupName": "npm updates"
+    },
+    {
+      "datasources": ["pip"],
+      "enabled": false
+    },
+    {
+      "depTypeList": ["go_repository"],
+      "enabled": false
+    }
+  ],
+  "ignorePaths": [
+    "components/deprecated/",
+    "third_party/",
+    "components/ibm-components/",
+    "components/azure/",
+    "components/presto/",
+    "components/arena/",
+    "backend/api/python_http_client/",
+    "manifests/kustomize/"
+  ],
+  "dependencyDashboard": true,
```

Comment:
```suggestion
```

This is redundant due to the previous inclusion of the `":dependencyDashboard"` preset and can be removed.